x-pack/plugins/cases/public/components/custom_fields/form_fields.test.tsx

```diff
@@ -56,9 +56,6 @@ describe('FormFields ', () => {
         {
           label: 'hello',
           type: 'toggle',
-          options: {
```

Comment:
Why it is not needed anymore?

Reply:
Oh I see. Probably because we removed the `defaultValue` from the `toggle/configure.tsx`. Make sense.